app/src/main/java/org/openobservatory/ooniprobe/item/TestsuiteItem.java

```diff
@@ -43,7 +43,12 @@ public TestsuiteItem(AbstractSuite extra, PreferenceManager pm, View.OnClickList
 		holder.itemView.setOnClickListener(onClickListener);
 		holder.run.setTag(extra);
 		holder.itemView.setTag(extra);
-		holder.runtime.setText(holder.runtime.getContext().getString(R.string.Dashboard_Card_Seconds, extra.getRuntime(pm).toString()));
+		//TODO convert seconds to minutes and hours when needed
+		//if getRuntime = 0 show one hour
+		Integer runTime = extra.getRuntime(pm);
+		if (runTime.equals(PreferenceManager.MAX_RUNTIME_DISABLED))
+			runTime = 3600;
+		holder.runtime.setText(holder.runtime.getContext().getString(R.string.Dashboard_Card_Seconds, runTime.toString()));
```

Comment:
Maybe we should factor this duplicate code?

Reply:
factored